src/tty.rs

```diff
@@ -235,6 +236,13 @@ pub fn new<T: ToWinsize>(config: &Config, options: &Options, size: T) -> Pty {
         Ok(())
     });
 
+    // Handle set working directory option
+    if let Some(ref dir) = options.working_dir {
+        env::set_current_dir(dir.as_path()).unwrap_or_else(|e| {
```

Comment:
Why not use [`std::process::Command::current_dir`](https://doc.rust-lang.org/1.2.0/std/process/struct.Command.html#method.current_dir)?